### ISSUE TITLE
GHVisualizer write names in buttons instead of only letters

### ISSUE THREAD

**Issue by christophwaibel** [architecture-building-systems/hive#341]
![image](https://user-images.githubusercontent.com/21927657/89310784-7b843100-d675-11ea-9c0d-7a7d39249080.png)

I have no idea what these letters stand for. Maybe better use short words instead?

**Comment by daren-thomas** [comment]
good point, but that will make the design fall apart. what if... tooltips?

**Comment by amraladdin** [comment]
@christophwaibel The meaning behind them are written in the document. I know it's not the best option atm, but writing the full name will change the design completely. I would use for now have an option where hovering with the mouse over the button shows the full name. Would that be an easy option to implement @daren-thomas ? Then we can redesign at a later stage.

**Comment by daren-thomas** [comment]
@amraladdin that totally depends on if `GH_Attributes` allows us to capture a hover event. i don't know yet.
